pb/config/config.proto

```diff
@@ -46,20 +55,24 @@ message Notification {
 
 // Specifies a group of tests to gather.
 message TestGroup {
-  reserved 5, 7, 8, 10, 13, 16 to 23, 28 to 36, 39 to 42, 44 to 51;
-
   // Name of this TestGroup, for mapping dashboard tabs to tests.
   string name = 1;
 
-  // Path to the test result stored in gcs
+  // Path to the test result location, including protocol.
+  // The location must be accessible from the Update Server, and not local.
+  // GCS is usually used, making this field "gs://<bucket>"
   string gcs_prefix = 2;
 
   // Number of days of test results to gather and serve.
   int32 days_of_results = 3;
 
-  // Whether to ignore pending (currently running) test results.
+  // Whether to ignore pending (currently running) test results. Recommended.
```

Comment:
"Recommended" as what? (Personally I'd recommend 'ignore_pending=false' usually since you can link to pending results).